pkg/graph/schema.resolvers.go

```diff
@@ -658,7 +654,18 @@ func (r *mutationResolver) DeleteAccessibilityRequestDocument(ctx context.Contex
 }
 
 func (r *queryResolver) AccessibilityRequest(ctx context.Context, id uuid.UUID) (*models.AccessibilityRequest, error) {
-	return r.store.FetchAccessibilityRequestByID(ctx, id)
+	accessibilityRequest, err := r.store.FetchAccessibilityRequestByID(ctx, id)
+	if err != nil {
+		return nil, err
+	}
+	ok, err := r.service.AuthorizeUserIs508TeamOrRequestOwner(ctx, accessibilityRequest)
+	if err != nil {
+		return nil, err
+	}
+	if !ok {
+		return nil, &apperrors.UnauthorizedError{Err: errors.New("unauthorized to delete accessibility request document")}
```

Comment:
This shouldn't be an unauthorized error - it should be a not found error. I know technically this is correct, but it's a security obfuscation to return a 404 instead of a 403.

Reply:
Also, the text here might be some copy/pasta?